packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -3043,7 +3043,7 @@ Content-Type: application/octet-stream
 --abcde12345--
 ```
 
-### Payload_MultiPart_FormData_fileWithHttpPartSpecificContentType
+### Payload_MultiPart_FormData_HttpPartTest_fileWithHttpPartSpecificContentType
```

Comment:
I also wouldn't use "specific", it's image jpeg, so it can be `Payload_MultiPart_FormData_HttpParts_ImageJpegContentType`.

We can also have a content types section, which has sub-ops `optional`, `required`, `image_jpeg`. But I'm not sure if all languages can handle nesting that far

Reply:
I group these 3 content type cases like non-string.